server/webdriver/cdc_tests/autopush_test.py

```diff
@@ -27,7 +28,7 @@
 CDC_AUTOPUSH_URL = 'https://dc-autopush-kqb7thiuka-uc.a.run.app'
 
 
-class CdcAutopushTest(unittest.TestCase):
+class CdcAutopushWebdriverTest(unittest.TestCase):
```

Comment:
@hqpho - With this PR, CDC tests now include non-webdriver tests as well. Wondering if we should still keep them in a `webdriver` folder or move them out of it?

Reply:
I see other NL tests are in server/integration_tests. Maybe it makes sense for all these tests to be in integration_tests? IMO that's a decent description of them!

Reply:
That works. Let's do this in a follow-on PR.